app/main.py

```
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.strip().split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+    _, src, dst = parts
+    if src == dst:
+        return
+    try:
+        with open(src, "r") as file_in, open(dst, "w") as file_out:
+            file_out.write(file_in.read())
+    except FileNotFoundError:
+        pass
```

According to the checklist, you may be required to print an error message when the source file is not found, rather than silently passing. Please check if the checklist requires user feedback for this error case.